debian/neo-go.install

```diff
@@ -0,0 +1,4 @@
+config/protocol.mainnet.yml etc/neo-go
+config/protocol.privnet.yml etc/neo-go
+config/protocol.testnet.yml etc/neo-go
```

Comment:
We also have a configuration for neofs storage node (mainnet/testnet). Although we don't have a separate CLI command to start the node with this config, it can still be useful for someone.

Reply:
All configs present in source code are copied to examples directory for reference.
These 3 files were installed in original Makefile so again here I'm trying to introduce as little changes as possible.